spyder/plugins/editor.py

```diff
@@ -976,7 +986,7 @@ def get_plugin_actions(self):
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 
         file_menu_actions = [self.new_action,
-                             None,
+                             self.open_last_closed,

```

Comment:
@goanpeca I just did that in this issue: https://github.com/spyder-ide/spyder/issues/3794